### PR TITLE
Refactor: simplify config arg attributes for keep_unsnapshoted_log

### PR DESCRIPTION
**Pull request by drmingdrmer**
##### Refactor: simplify config arg attributes for keep_unsnapshoted_log

---
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/openraft/465)

### CODE REVIEW

**Comment by drmingdrmer** [comment]
@ClSlaid

**Comment by ClSlaid** [comment]
Looks good to me, but document comments should be slightly changed.
"set this flag to true" -> "apply this flag".

File: openraft/src/config/config.rs [hunk 1]
```diff
@@ -130,7 +130,12 @@ pub struct Config {
     ///
     /// By default, `OpenRaft` purges `applied_log`s from time to time regardless of snapshots, because it assumes once
     /// logs are `applied` to the state machine, logs are persisted on disk. If not so, set this to true.
```
Comment (ClSlaid) [inline openraft/src/config/config.rs:132]: > set this to true

maybe "apply this flag" is better.

Comment (drmingdrmer) [reply openraft/src/config/config.rs]: Either would be fine to me. But the explanation should be elaborated. Have a look at my next commit.

**Pull request merged**